behaviors/wysiwyg.scss

```diff
@@ -54,10 +55,57 @@ $wysiwyg-dropshadow: 0 0 6px $input-shadow;
   top: -8px;
 }
 
+// two arrows, since the first goes over the border of the toolbar itself
+// and the second goes under and applies the drop shadow
+// .medium-toolbar-arrow-under:before {
+//   border-color: $wysiwyg-bgcolor transparent transparent transparent;
+//   border-style: solid;
+//   border-width: 8px 8px 0;
+//   content: '';
+//   display: block;
+//   height: 0;
+//   left: 50%;
+//   margin-left: -8px;
+//   position: absolute;
+//   top: $wysiwyg-button-size;
+//   width: 0;
+//   z-index: 1;
+// }
+//
+// .medium-toolbar-arrow-under:after {
+//   border-color: $wysiwyg-bgcolor transparent transparent transparent;
+//   filter: drop-shadow($wysiwyg-dropshadow);
+//   top: $wysiwyg-button-size;
+//   z-index: -1;
+// }
+
+.medium-toolbar-arrow-under:before,
 .medium-toolbar-arrow-under:after {
-  border-color: $wysiwyg-bgcolor transparent transparent transparent;
-  filter: drop-shadow($wysiwyg-dropshadow);
+	border: 1px solid transparent;
```

Comment:
Meant that the indentations look inconsistent -- some are tabs and some are spaces.